packaging/debian/preinst

```diff
@@ -4,6 +4,12 @@ if [ ! -d /run/percona-telemetry-agent ]; then
     install -m 0755 -d -oroot -groot /run/percona-telemetry-agent
 fi
 
+# Create new linux group
+groupadd percona-telemetry
```

Comment:
shall we delete this group during package removing?

Reply:
Oh Yes, I shall add command to delete the group during package removal in postrm and spec file.